tests/ArrayTarget.php

```diff
@@ -7,22 +7,21 @@
 
 namespace Yiisoft\Log\Tests;
 
-use yii\exceptions\Exception;
 use Yiisoft\Log\Target;
 
 /**
  * ArrayTarget logs messages into an array, useful for tracking data in tests.
  */
 class ArrayTarget extends Target
 {
-    public $exportInterval = 1000000;
+    protected $exportInterval = 1000000;
```

Comment:
Could be private, I think.

Reply:
This wouldn't work as it won't impact Target properties.
Changed to using the setter in __construct()